tools/asr_evaluator/conf/eval.yaml

```diff
@@ -12,8 +12,9 @@ engine:
         mode: offline # choose from offline, chunked or offline_by_chunked
         chunk_len_in_secs: 1.6 #null # Need to specify if use buffered inference (default for offline_by_chunked is 20)
         total_buffer_in_secs: 4 #null # Need to specify if use buffered inference (default for offline_by_chunked is 22)
-        model_stride: 4 # Model downsampling factor, 8 for Citrinet models and 4 for Conformer models
+        model_stride: 8 # Model downsampling factor, 8 for Citrinet and FastConformer models, and 4 for Conformer models
         decoder_type: null # Used for hybrid CTC RNNT model only. Specify decoder_type *ctc* or *rnnt* for hybrid CTC RNNT model.
+   
     test_ds:
         manifest_filepath: null
         sample_rate: 16000
```

Comment:
Please add `num_workers` here and also pass it in `https://github.com/NVIDIA/NeMo/blob/main/tools/asr_evaluator/utils.py#L118`